cli_helpers/tabular_output/tabulate_adapter.py

```diff
@@ -3,16 +3,17 @@
 
 from cli_helpers.packages import tabulate
 from cli_helpers.utils import filter_dict_by_key
-from .preprocessors import (bytes_to_string, align_decimals,
-                            override_missing_value)
+from .preprocessors import (convert_to_string, align_decimals,
```

Comment:
Looks like the preprocessors are already in the right order, actually. So, I think we're good here.

The ones that operate based on field type go first, then the string conversion, and finally styling the output.